### PR TITLE
Rework includes possible suggestion for bsd

### PR DESCRIPTION
**Pull request by splbio**

### CODE REVIEW

**Comment by splbio** [comment]
oops, my git is a bit rusty, let me redo this...

File: src/libprojectM/Renderer/CMakeLists.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ endif(MSVC)
 
 if (APPLE)
 	FIND_LIBRARY(COREFOUNDATION_LIBRARY CoreFoundation)
-	INCLUDE_DIRECTORIES( /usr/include )
+	INCLUDE_DIRECTORIES( /usr/include /opt/local/include )
```
Comment (revmischa) [inline src/libprojectM/Renderer/CMakeLists.txt]: okay all the cmake files need to die, we're using autotools now

File: src/projectM-emscripten/projectM_SDL_emscripten.cpp [hunk 2]
```diff
@@ -190,7 +190,7 @@ int main( int argc, char *argv[] ) {
 
     // mainloop. non-emscripten version here for comparison/testing
 #ifdef EMSCRIPTEN
-    emscripten_set_main_loop(renderFrame, 0, 0);
+    emscripten_set_main_loop((void (*)())renderFrame, 0, 0);
```
Comment (revmischa) [inline src/projectM-emscripten/projectM_SDL_emscripten.cpp]: i think you're on the wrong branch?

**Comment by splbio** [comment]
> oops, my git is a bit rusty, let me redo this...

**Pull request closed without merging**